tool/elk/startup/startup-elk.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -ex
+
+sudo gcloud secrets versions access latest --secret certificate | base64 -d | sudo tee /etc/elasticsearch/cert.pem
+sudo gcloud secrets versions access latest --secret key | base64 -d | sudo tee /etc/elasticsearch/key.pem
+
+sudo chmod a+r /etc/elasticsearch/cert.pem
+sudo chmod a+r /etc/elasticsearch/key.pem
+sudo chmod a+x /etc/elasticsearch/
```

Comment:
Kibana would need to read `cert.pem` and `key.pem`. In order for file to be accessible, not only should you have read permissions on the file itself, but also have exceute permissions on all of the parent directories. Therefore, we mark `/etc/elasticsearch` as accesible to be listed ("executable" bit) for all users.